chia/data_layer/s3_plugin_service.py

```diff
@@ -87,6 +89,12 @@ async def upload(self, request: web.Request) -> web.Response:
             return web.json_response({"handles_url": False})
         return web.json_response({"uploaded": True})
 
+    async def plugin_id(self, request: web.Request) -> web.Response:
+        return web.json_response({"id": plugin_id, "version": plugin_version})
```

Comment:
Should `"success": True` be included everywhere that we respond successfully?

Reply:
no